dom0/sd-clean-whonix.sls

```diff
@@ -14,7 +14,6 @@ remove-securedrop-log-package-from-whonix:
 sd-cleanup-whonix-gw-15:
   cmd.run:
     - names:
-      - sudo rm -f /etc/rsyslog.d/sdlog.conf
```

Comment:
Note that we use `pkg.purged` to remove the `securedrop-log` package, which [removes config files](https://docs.saltstack.com/en/latest/ref/states/all/salt.states.pkg.html):

> Verify that a package is not installed, calling pkg.purge if necessary to purge the package. All configuration files are also removed.

Removing a configuration file on a Debian system by other means can have unpredictable consequences: the package manager will not reinstate it , because it will assume that the user removed it intentionally. For this reason, I think it's best to rely on `pkg.purged` alone here.